pkg/daemon/ceph/client/crush_rule.go

```diff
@@ -150,8 +150,9 @@ func buildTwoStepCrushSteps(pool cephv1.PoolSpec) []stepSpec {
 	// Step three
 	stepTakeSubFailureDomain := &stepSpec{
 		Operation: "chooseleaf_firstn",
-		Number:    int(pool.Replicated.ReplicasPerFailureDomain),
-		Type:      pool.Replicated.SubFailureDomain,
+		// nolint:gosec // G115 No overflow
```

Comment:
I'd like to have clear explanations in linter skip comments that explain clearly why the decision was made, to allow easier reading and code changes in the future

```suggestion
		// nolint:gosec // G115 - casting uint to int has no overflow
```